services/apps/alcs/src/providers/typeorm/migrations/1687810457499-seed_inactive_local_gov.ts

```diff
@@ -33,13 +34,13 @@ export class seedInactiveLocalGov1687810457499 implements MigrationInterface {
 	 ('dc0357b1-aacd-45ba-8fa9-0e62cbe88e9f',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust Salt Spring Island (Historical)','ISLR',NULL,false,false,'{}'),
 	 ('3890ccf8-94e3-4b9a-9a1c-4d33518e1b5d',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust Saturna Island (Historical)','ISLR',NULL,false,false,'{}'),
 	 ('538b895f-f0e1-4388-93cb-9eeb7cfa987e',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust Sidney Island (Historical)','ISLR',NULL,false,false,'{}'),
-	 ('88532d18-3059-4720-986a-363d9bce6be0',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust- Comox Strathcona (Historical)','ISLR',NULL,false,false,'{}'),
-	 ('4e721d39-09b3-4f9b-ad1f-601a69315808',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust- Nanaimo (Historical)','ISLR',NULL,false,false,'{}'),
-	 ('9355bf78-280a-4695-adad-fafa06969c99',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust-Capital (Historical)','ISLR',NULL,false,false,'{}'),
-	 ('8017df89-8e71-41c3-bee9-1a48f042e157',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust-Powell River (Historical)','ISLR',NULL,false,false,'{}'),
-	 ('26f395a1-44ab-4812-9e0d-f2dbc815859d',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust-Sunshine Coast (Historical)','SOUR',NULL,false,false,'{}'),
-	 ('b9056ee1-5ba3-4d7c-9029-2cec57ba75d5',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust - Cowichan Valley (Historical)','ISLR',NULL,false,false,'{}'),
-	 ('55f665bc-c91b-4bbb-85c2-39691088b297',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust Bowen Island (Historical)','SOUR',NULL,false,false,'{}'),
+	 ('88532d18-3059-4720-986a-363d9bce6be0',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust Comox Strathcona (Historical)','ISLR',NULL,false,false,'{}'),
+	 ('4e721d39-09b3-4f9b-ad1f-601a69315808',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust Nanaimo (Historical)','ISLR',NULL,false,false,'{}'),
+	 ('9355bf78-280a-4695-adad-fafa06969c99',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust Capital (Historical)','ISLR',NULL,false,false,'{}'),
+	 ('8017df89-8e71-41c3-bee9-1a48f042e157',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust Powell River (Historical)','ISLR',NULL,false,false,'{}'),
+	 ('26f395a1-44ab-4812-9e0d-f2dbc815859d',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust Sunshine Coast (Historical)','SOUR',NULL,false,false,'{}'),
+	 ('b9056ee1-5ba3-4d7c-9029-2cec57ba75d5',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust Cowichan Valley (Historical)','ISLR',NULL,false,false,'{}'),
+	 ('55f665bc-c91b-4bbb-85c2-39691088b297',NULL,NOW(),NULL,'migration_seed',NULL,'Northern Rockies (Historical)','NORR',NULL,false,false,'{}'),
```

Comment:
updating the migration will not affect already inserted data is that intentional?

Reply:
Fixed, should have been in a previous MR. Will create new migration later

Reply:
I would prefer creating a new migration now or leaving out these changes, it would be confusing to look at migration history and see a migration that fixes nothing compared to old migrations.

Reply:
migration reverted and removed from MR